src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Aes.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.Intrinsics;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    /// <summary>
+    /// This class provides access to Intel AES hardware instructions via intrinsics
+    /// </summary>
+    [CLSCompliant(false)]
+    public static class Aes 
+    {
+        public static bool IsSupported { get { return false; } }
```

Comment:
Looks like all of the Api is always throwing PlatformNotSupported except for this member on all classes. @weshaggard is it possible to generate all of the PNSE code through a property on the csproj? I know that it is possible to do it for the whole assembly, but I wonder if it's possible to do so for everything BUT this api which should return false?

Reply:
This approach looks pretty cool. But if it is difficult, I think we can just leave the current PNSE code. This C# code won't be changed anymore.

Reply:
I think it is fine to have the .cs file. (BTW: We have the same file in CoreLib as well.)